.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+language: node_js
+node_js:
+  - stable
```

Comment:
You sure we should stick with `stable`. It's very fresh despite its name. 
`lts` would be the safer choice, imho